vagrant/k8s/Vagrantfile

```diff
@@ -7,14 +7,6 @@ require 'fileutils'
 # netplugin_synced_gopath="/opt/golang"
 gopath_folder="/opt/gopath"
 
-# method to set up attributes for better performance
-def customize_vm(v)
-  mem = `grep 'MemTotal' /proc/meminfo | sed -e 's/MemTotal://' -e 's/ kB//'`.to_i / 1024 / 8
```

Comment:
Can we skip this only if /proc/meminfo is unavailable? That way, on linux servers, this will run faster.